xsensemulator/emulator.go

```diff
@@ -73,7 +74,7 @@ type Emulator struct {
 	lastMessageIdentifier xsens.MessageIdentifier
 }
 
-func NewEmulator(p xsens.SerialPort) *Emulator {
+func NewEmulator(p io.ReadWriteCloser) *Emulator {
```

Comment:
Hmm I'm a bit confused, isn't the emulator just for running over UDP? If that is the case I guess we could take a UDPSerialPort as arg?